src/trezor.js

```diff
@@ -105,12 +105,16 @@ export const TREZOR_BOTH_BUTTONS = 'trezor_both_buttons';
  */
 export const TREZOR_PUSH_AND_HOLD_BUTTON = 'trezor_push_and_hold_button';
 
+const TREZOR_CONNECT_URL = "https://localhost:8088/";
+const TREZOR_BLOCKBOOK_URL = "http://localhost:3035"
```

Comment:
can we put these in process env too and have these values as fallbacks?

Reply:
good call